airbyte-webapp/src/pages/connections/ConnectionStatusPage/ConnectionStatusPage.tsx

```diff
@@ -162,35 +162,31 @@ export const ConnectionStatusPage: React.FC = () => {
         title={
           <div className={styles.title}>
```

Comment:
`styles.title` should have a min-height that matches the button's height. Currently, when the buttons disappear, the height of the title changes. Ideally, there is no height shift.